app/about.html

```diff
@@ -33,10 +38,13 @@
         />
       </a>
       <div class="nav-links">
+        <a href="javascript:void(0);" class="close-icon">
+          <i class="fa fa-bars" aria-hidden="true"></i>
```

Comment:
Asking because I genuinely don't know the answer: do we need an alt-tag or something accessibility/screen-reader friendly that labels this as "Menu"?

Reply:
Same question for when this swaps out for the "Close" x.